samoa-api/build/test/data/dfs/data/data1/current/BP-62506341-193.10.66.77-1421760460444/current/VERSION

```diff
@@ -0,0 +1,5 @@
+#Tue Jan 20 14:27:40 CET 2015
```

Comment:
These files should not be checked in (actually, we should remove them automatically after the tests finish).